doc/engineering-reference/src/daylighting-and-window-calculations/window-calculation-module.tex

```diff
@@ -288,33 +288,97 @@ \subsubsection{Step 6.~ Determine Layer Visible Properties}\label{step-6.-determ
+
+\begin{equation}
+  R(\phi) = R(0)\rho_{old}(\phi)
+\end{equation}
+
+to the form used for coated glazing,
+
+\begin{equation}
+  R(\phi ) = R(0)(1 - \rho_{new}(\phi )) + \rho_{new}(\phi )
+\end{equation}
+
+set them equal to each other and solve algebraically for $\rho_{new}(\phi )$:
+
+\begin{equation}
+  \rho_{new}(\phi ) = \frac{R(0)\left(\rho_{old}(\phi ) - 1\right)}{1 - R(0)}
+\end{equation}
+
+where,
+
+\begin{equation}
+  \rho_{new}(\phi ) = a_{new} + b_{new}\cos (\phi ) + c_{new}{\cos ^2}(\phi ) + d_{new}{\cos ^3}(\phi ) + e_{new}{\cos ^4}(\phi ) - \tau (\phi )
+\end{equation}
+
+and,
+
+\begin{equation}
+  \rho_{old}(\phi ) = a_{old} + b_{old}\cos (\phi ) + c_{old}{\cos ^2}(\phi ) + d_{old}{\cos ^3}(\phi ) + e_{old}{\cos ^4}(\phi )
+\end{equation}
+
+The updated coefficient values for a, b, c, d, and e are listed for transmittance and reflectance in Tables \ref{tab:normalized-transmittance-correlations-for} and \ref{tab:normalized-reflectance-correlations-for}, respectively.
+
+\begin{table}[tbp]
+  \caption{Normalized Transmittance Correlations for Angular Performance}
+  \label{tab:normalized-transmittance-correlations-for}
+  \centering
+  \begin{tabular}{llllll}
+  \hline
+  Curve                                 & a    & b    & c     & d     & e    \\ \hline
+  A - Single: 3mm clear                 & 0.00 & 3.36 & -3.85 & 1.49  & 0.01 \\
+  B - Single: 3mm bronze                & 0.00 & 2.83 & -2.42 & 0.04  & 0.55 \\
+  C - Single: 6mm bronze                & 0.00 & 2.45 & -1.58 & -0.64 & 0.77 \\
+  D - Single: 3mm coated                & 0.00 & 2.85 & -2.58 & 0.40  & 0.35 \\
+  E - Double: 3mm clear, clear          & 0.00 & 1.51 & 2.49  & -5.87 & 2.88 \\
+  F - Double: 3mm coated, clear         & 0.00 & 1.21 & 3.14  & -6.37 & 3.03 \\
+  G - Double: 3mm tinted, clear         & 0.00 & 1.09 & 3.54  & -6.84 & 3.23 \\
+  H - Double: 6mm coated, clear         & 0.00 & 0.98 & 3.83  & -7.13 & 3.33 \\
+  I - Double: 6mm tinted, clear         & 0.00 & 0.79 & 3.93  & -6.86 & 3.15 \\
+  J - Triple: 3mm coated, clear, coated & 0.00 & 0.08 & 6.02  & -8.84 & 3.74
+  \end{tabular}
+\end{table}
+
+\begin{table}[tbp]
+  \caption{Normalized Reflectance Correlations for Angular Performance}
+  \label{tab:normalized-reflectance-correlations-for}
+  \centering
+  \begin{tabular}{llllll}
+  \hline
+  Curve                                 & a    & b     & c     & d      & e    \\ \hline
+  A - Single: 3mm clear                 & 1.00 & -0.70 & 2.57  & -3.20  & 1.34 \\
+  B - Single: 3mm bronze                & 1.00 & -1.87 & 6.51  & -7.86  & 3.23 \\
+  C - Single: 6mm bronze                & 1.00 & -2.53 & 8.40  & -9.86  & 3.99 \\
+  D - Single: 3mm coated                & 1.00 & -1.86 & 6.40  & -7.64  & 3.10 \\
+  E - Double: 3mm clear, clear          & 1.00 & -1.57 & 5.60  & -6.83  & 2.80 \\
+  F - Double: 3mm coated, clear         & 1.00 & -3.14 & 10.97 & -13.14 & 5.32 \\
```

Comment:
In the code -3.14 is -3.15. Similar differences on the same coefficient for G and I. Also some minor differences in coefficients c, d, and e for several rows.